packages/scandipwa/src/route/ProductPage/ProductPage.style.scss

```diff
@@ -31,6 +31,15 @@
         grid-template-rows: min-content min-content;
         padding-block-start: 24px;
 
+        @include after-mobile {
+            grid-template-areas: '. actions';
+            padding-bottom: 7.2rem;
```

Comment:
Please use px.

Reply:
And use logical css properties. Like `padding-block-end` instead of `padding-bottom`.